project/src/main/webapp/discovery-style.css

```diff
@@ -0,0 +1,82 @@
+.header{
+  padding: 30px;
+  text-align: center;
+  background: white;
+}
+
+.subheader {
+  padding: 50px;
+  text-align: center;
+}
+
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.content-container {
+    text-align: center;
+}
+
+.profile-header {
+  padding: 20px;
+  margin-top: 20px;
+}
+
+.profile-header__left-column {
+  float: left;
+  width: 25%;
+}
+
+.profile-header__right-column{
+  float: left;
+  width: 75%;
+}
+
+.profile-header__bio {
+  float: left;
+  text-align: left;
+  padding: 30px;
+  border-color: black;
+  width: 75%;
+}
+
+img {
+    border-radius: 50%;
+    width: 75px;
+    height: 75px;
+}
+
+#myInput {
+  background-image: url(''); /* Add a search icon to input */
+  background-position: 10px 12px; /* Position the search icon */
+  background-repeat: no-repeat; /* Do not repeat the icon image */
+  width: 100%; /* Full-width */
+  font-size: 16px; /* Increase font-size */
+  padding: 12px 20px 12px 40px; /* Add some padding */
+  border: 1px solid #ddd; /* Add a grey border */
+  margin-bottom: 12px; /* Add some space below the input */
+}
```

Comment:
A lot of these comments seem to just be describing the standard function of the CSS properties --- are they all needed?

Reply:
+1. Recommend deleting the comments.

Reply:
Nope they're not needed, I'll remove them.